build-system/tasks/e2e/describes-e2e.js

```diff
@@ -410,7 +410,8 @@ function describeEnv(factory) {
       this.timeout(TEST_TIMEOUT);
       beforeEach(async function() {
         this.timeout(SETUP_TIMEOUT);
-        await fixture.setup(env, browserName);
+        const retries = 1;
```

Comment:
Move this to the list of constants at the top of the file.

Reply:
Done.